music21/metadata/primitives.py

```diff
@@ -805,12 +871,51 @@ class Copyright(Text):
     'fictitious'
     >>> str(copyleft)
     'Copyright 1969 Cuthbert'
+    >>> copyleft2 = metadata.Copyright('Copyright 1969 Cuthbert',
+    ...                 role='fictitious')
```

Comment:
indentation.   (ack.  there were problems with indentation before.  My bad.)

If possible maybe can remove my name from all these examples -- I didn't like it there before but now that it appears 10 more times, I really don't like it.  :-D